docs/admin/security/connection-logs.md

```diff
@@ -0,0 +1,103 @@
+# Connection Logs
+
+The Connection Log page in the dashboard allows **Auditors** to monitor workspace agent connections.
+
+## Workspace app connections
+
+The connection log contains a complete record of all **workspace app** connections.
+These originate from within the Coder deployment, and thus the connection log
+is a source of truth for these events.
+
+## Browser port forwarding
+
+The connection log contains a complete record of all workspace port forwarding
+performed via the web dashboard.
+
+## SSH & IDE sessions
+
+The connection log aims to capture a record of SSH & IDE sessions to workspaces.
+These events are reported by workspace agents, and their receipt by the server
+is not guaranteed.
+
+## Filtering logs
+
+Connection logs can be filtered with the following parameters:
+
+- `organization` - The name or ID of the organization of the workspace being
+     connected to.
+- `workspace_owner` - The username of the owner of the workspace being connected
+    to.
+- `type` - The type of the connection (i.e. SSH, VSCode, Workspace App).
+    An exhaustive list is
+    [available here](https://pkg.go.dev/github.com/coder/coder/v2/codersdk#ConnectionType).
+- `username`: The name of the user who initiated the connection. Results will
+   not include SSH & IDE sessions.
+- `user_email`: The email of the user who initiated the connection. Results will
+   not include SSH & IDE sessions.
+- `started_after`: The time after which the connection started. Uses the RFC3339Nano format.
+- `started_before`: The time before which the connection started. Uses the RFC3339Nano format.
+- `workspace_id`: The ID of the workspace being connected to.
+- `connection_id`: The ID of the connection.
+- `status`: The status of the connection, either `ongoing` or `completed`.
+     Some events are neither ongoing nor completed, such as the opening of a
+     workspace app.
+
+## Capturing/Exporting Connection Logs
+
+In addition to the user interface, there are multiple ways to consume or query
+connection events.
+
+## REST API
+
+Connection logs can be retrieved via our REST API. You can find detailed
+information about this in our
+[endpoint documentation](../../reference/api/enterprise.md#get-connection-logs).
```

Comment:
```suggestion
You can retrieve connection logs via the Coder API.

Visit the
[`get-connection-logs` endpoint documentation](../../reference/api/enterprise.md#get-connection-logs)
for details.
```

I think we call it the `Coder API` more than `REST API` - the heading title can probably stay though